### PR TITLE
[RA1 Ch02] Update chapter02.md

### PR DESCRIPTION
**Pull request by zwarico**
Recommended changes to section 2.3.8.2. Platform and Access

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 1]
```diff
@@ -197,19 +197,19 @@ Traceability to Reference Model.
...
-| sec.sys.004 | Access | The Cloud Infrastructure **must** support Secure network channels | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
-| sec.sys.005 | Access | The Cloud Infrastructure **must** segregate the underlay and overlay networks | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
-| sec.sys.006 | Access | The Cloud Infrastructure **must** be able to utilize the Cloud Infrastructure Manager identity management capabilities | [RA-1 6.3.1 "Identity"](./chapter06.md#6311-identity) |
-| sec.sys.007 | Access | The Platform **must** implement controls enforcing separation of duties and privileges, least privilege use and least common mechanism (Role-Based Access Control) | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
+| sec.sys.001 | Access | The Platform **must** support authenticated and secure access to API, GUI and command line interfaces | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:200]: For consistency:
```suggestion
| sec.sys.001 | Access | The Platform **must** support authenticated and secure access to API, GUI and command line interfaces. | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
```

File: doc/ref_arch/openstack/chapters/chapter02.md [hunk 2]
```diff
@@ -197,19 +197,19 @@ Traceability to Reference Model.
...
+| sec.sys.010 | Access | For two or more domains without existing trust relationships, the Platform **must not** allow the effect of an attack on one domain to impact the other domains either directly or indirectly. | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
 | sec.sys.011 | Access | The Platform **must not** reuse the same authentication key-pair (for example, on different hosts, for different services) | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
 | sec.sys.012 | Access | The Platform **must** only use secrets encrypted using strong encryption techniques, and stored externally from the component (e.g., Barbican (OpenStack)) | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
-| sec.sys.013 | Access | The Platform **must** provide secrets dynamically as and when needed | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
+| sec.sys.013 | Access | The Platform **must** generate secrets dynamically as and when needed | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter02.md:212]: For consistency:
```suggestion
| sec.sys.013 | Access | The Platform **must** generate secrets dynamically as and when needed. | [RA-1 6.3.1 "Platform Access"](./chapter06.md#631-platform-access) |
```

File: doc/ref_arch/openstack/chapters/chapter02.md (hunk 1 above)
Comment (zwarico) [reply doc/ref_arch/openstack/chapters/chapter02.md:200]: Agreed. Thanks Pankaj.

**Review by walterkozlowski — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@pgoyal01: can you please approve and merge?

**Pull request merged**